src/controllers/wallet.js

```diff
@@ -0,0 +1,56 @@
+var Validator = require('validator');
+var Restify = require('restify');
+
+var WalletModel = require('../models/wallet');
+var Config = require('../config');
+
+var wallet = {
+    /**
+     * View method for POST /v1/wallet
+     *
+     * Body arguments (JSON or urlencoded)
+     * msisdn - currently limited to one locale (set in Config.VALIDATION_COUNTRY)
+     * pin - integer at least 5 digits
+     */
+    create: function(req, res, next) {
+        // NOTE: request must have content-type set application/json 
+        // or application/x-www-form-urlencoded
+        var error_message = '', msisdn, pin;
+        if (req.params) {
+            msisdn = req.params['msisdn'];
+            pin = req.params['pin'];
```

Comment:
Might we worthwhile pulling in something like `lodash` here so we can more easily specify defaults, check for missing keys, etc.